### PR TITLE
Otterspace - new networks [otterspace-badges]

### PR DESCRIPTION
**Pull request by rahulrumalla**
1. Adds 2 new neworks - Mainnet & Optimism goerli
2. Udpates logic for using the badge with the highest weight when an address holds multiple badges
3. Updates exmaples' address to using checksums

### CODE REVIEW

File: src/strategies/otterspace-badges/index.ts [hunk 1]
```diff
@@ -5,8 +5,11 @@ export const author = 'otterspace-xyz';
 export const version = '1.0.0';
```
Comment (ChaituVR) [inline src/strategies/otterspace-badges/index.ts:5]:
```suggestion
export const version = '1.0.1';
```

File: src/strategies/otterspace-badges/index.ts [hunk 2]
```diff
@@ -75,9 +79,17 @@ function applyBadgeWeights(badges: [], options: any) {
...
-    badgeWeights[badgeAddress] = getBadgeWeight(options.specs, badge.spec.id);
+    // picks the highest weight when multiple badges are held by the same address
+    if (
+      !badgeWeights[badgeAddress] ||
+      badgeWeight > badgeWeights[badgeAddress]
+    ) {
+      badgeWeights[badgeAddress] = badgeWeight;
+    } else {
+      return;
+    }
```
Comment (ChaituVR) [inline src/strategies/otterspace-badges/index.ts:92]: Hi @rahulrumalla There is a active proposal on cowgrants.eth space, that is using this space, can this change create a problem in someone's voting power? better to wait until the proposal is closed

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/otterspace-badges/index.ts (hunk 2 above)
Comment (ChaituVR) [reply src/strategies/otterspace-badges/index.ts:92]: oh i think it may not cause a problem looking at the code, can confirm? :)

Comment (rahulrumalla) [reply src/strategies/otterspace-badges/index.ts:92]: It shouldn't cause a problem

**Review by ChaituVR — APPROVED** [review]

**Comment by rahulrumalla** [comment]
> @rahulrumalla Test cases are failing, can take a look? :)

Done!

**Pull request merged**